sdks/python/apache_beam/runners/portability/fn_api_runner_test.py

```diff
@@ -47,11 +50,31 @@
   DEFAULT_SAMPLING_PERIOD_MS = 0
 
 
+def suppress_stderr(f):
```

Comment:
Naming nit:
Following nose's naming conventions (--nocapture, --nologcapture), I'd call this wrapper `capture_stderr`.

Reply:
Also, we could alternatively remove --nocapture, --nologcapture flags from unit tests, which should have the same effect.

Reply:
Changed. I also rebased due to merge conflicts. 

Having nocapture on in general is quite useful for tests that hang. I looked around and didn't see a way to control it programatically per test.